src/components/chatbot/ChatBox.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import ChatBoxForm from './ChatBoxForm';
+import ChatMessage from './ChatMessage';
+import { useChat } from '@/hooks/useChat';
+import { useSendMessage } from '@/hooks/useBot';
+import { ByUser, Message } from '@/types/messages/message';
+import { getMessagesFromChat } from '@/lib/chat';
+
+type P = {
+  // ID of the chat to retrieve history of messages from. Not the TG Chat ID, but the actual chat Id stored in database
+  chatId: number;
+}
+
+export default function ChatBox({chatId}: P) {
+  const {data: chat, isPending} = useChat(chatId);
+  const sendMessageMutation = useSendMessage(chatId)
+
+  const [messages, setMessages] = useState<Message[]>([])
+  
+  useEffect(() => {
+    if (chat) {
+      setMessages(getMessagesFromChat(chat));
+    }
+
+  }, [chat])
+  
+  useEffect(() => {
+    try {
+      // TODO: Replace URL with constant string
+      const socket = new WebSocket('ws://localhost:8080/api/ws');
```

Comment:
```suggestion
      const socket = new WebSocket(process.env.NEXT_PUBLIC_BACKEND_WS_URL);
```

Try to have (base) urls as environment variables and save them as constants at build time.

Reply:
Does this function instead make more sense?
```
export const MESSAGE_WS_API_URL = () => {
    const url = new URL(process.env.NEXT_PUBLIC_BACKEND_URL!);
    // host = localhost:8080
    // pathname = /api
    const wsUrl = `ws://${url.host}${url.pathname}/ws`;
    return wsUrl;
}
```

Reply:
I think that approach can work. We can rename the function as a getter (getWebSoccketUrl)  to avoid confusion then 👍🏻